README.md

```diff
@@ -560,11 +560,14 @@ A curated list of awesome Java frameworks, libraries and software.
 
 ## Science
 
-*Libraries for scientific computing and analysis.*
+*Libraries for scientific computing, analysis and visualization.*
 
 * [DataMelt](http://jwork.org/dmelt/) - Environment for scientific computation, data analysis and data visualization.
-* [JGraphT](https://github.com/jgrapht/jgrapht) - Graph library that provides mathematical graph-theory objects and algorithms.
 * [JScience](http://jscience.org/) - Provides a set of classes to work with scientific measurements and units.
+* [GraphStream](http://graphstream-project.org/) - GraphStream is a Java library for the modeling and analysis of dynamic graphs. You can generate, import, export, measure, layout and visualize them.
+* [JGraphT](https://github.com/jgrapht/jgrapht) - Graph library that provides mathematical graph-theory objects and algorithms.
+* [JGraphX](https://github.com/jgraph/jgraphx) - JGraphX provides functionality for visualisation and interaction with node-edge graphs (not charts).
```

Comment:
The main usage is in Swing applications isn't it? Why not add it to the description?